docs/resources/ocean_aws.md

```diff
@@ -304,6 +304,8 @@ autoscaler {
 * `update_policy` - (Optional)
     * `should_roll` - (Required) Enables the roll.
     * `conditioned_roll` - (Optional, Default: false) Spot will perform a cluster Roll in accordance with a relevant modification of the cluster’s settings. When set to true , only specific changes in the cluster’s configuration will trigger a cluster roll (such as AMI, Key Pair, user data, instance types, load balancers, etc).
+    * `conditioned_roll_params` - (Optional) List of string which gives the user, privilege to input a customized list of attributes upon modification of which in the configuration will trigger the cluster roll. (Valid values: `"subnet_ids"`, `"whitelist"`, `"blacklist"`, `"user_data"`, `"image_id"`, `"security_groups"`,
```

Comment:
When modified, a custom list of attributes will trigger the cluster roll operation (overrides the predefined list of parameters). Valid only when the `conditioned_roll` parameter is set to true.

Reply:
updated